ZipBuilder/Sources/FirebaseReleaser/InitializeRelease.swift

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+import FirebaseManifest
+import Utils
+
+struct InitializeRelease {
+  static func setupRepo(gitRoot: URL) -> String {
+    let manifest = FirebaseManifest.shared
+    let branch = createReleaseBranch(path: gitRoot, version: manifest.version)
+    updatePodspecs(path: gitRoot, manifest: manifest)
+    updatePodfiles(path: gitRoot, version: manifest.version)
+    return branch
+  }
+
+  /// The branch is based on the minor version to represent this is the branch for subsequent
+  /// patches.
+  private static func createReleaseBranch(path: URL, version: String) -> String {
+    let versionParts = version.split(separator: ".")
+    let minorVersion = "\(versionParts[0]).\(versionParts[1])"
+    let branch = "release-\(minorVersion)"
+    Shell.executeCommand("git checkout master", workingDir: path)
+    Shell.executeCommand("git pull", workingDir: path)
+    Shell.executeCommand("git checkout -b \(branch)", workingDir: path)
+    return branch
+  }
+
+  /// Update the podspec versions.
+  private static func updatePodspecs(path: URL, manifest: FirebaseManifest.Manifest) {
+    for pod in manifest.pods {
+      if !pod.isClosedSource {
+        if pod.name == "Firebase" {
+          updateFirebasePodspec(path: path, manifest: manifest)
+        } else {
+          let version = pod.podVersion ??
+            (pod.isBeta ? manifest.version + "-beta" : manifest.version)
+          Shell.executeCommand("sed -i.bak -e \"s/\\(\\.version.*=[[:space:]]*'\\).*'/\\1" +
```

Comment:
I would appreciate a brief command on what this command is doing.

Reply:
Done.